papers/hendrik_heuer/heuer.rst

```diff
@@ -0,0 +1,223 @@
+
+The general idea behind word space models is to use distributional statistics to generate high-dimensional vector spaces, where a word is represented by a context vector that encodes semantic similarity [Sah05]. The representations are called distributed representations because the features are not mutually exclusive and because their configurations correspond to the variations seen in the observed data [Cun15]. LeCun et al. provide the example of a news story. When the task is to predict the next word in a news story, the learned word vectors for Tuesday and Wednesday will be very similar as they can be easily replaced by each other when used in a sentence [Cun15].
+
+There is a variety of computational models that implement the distributional hypothesis, including word2vec [Che13], GloVe [Pen14], dependency-based word embeddings [Lev14] and Random Indexing [Sah05]. For all of the techniques, Python implementations exist. word2vec is available in gensim [Řeh10]. For GloVe, the C source code was ported to Python [Gau15, Kul15]. The dependency-based word embeddings by Levy and Goldberg are implemented in spaCy [Hon15]. Random Indexing is available in an implementation by Joseph Turian [Tur15].
+
+For this paper, word2vec was selected because Mikolov et al. provide 1.4 million pre-trained entity vectors trained on 100 billion words from various news articles in the Google News dataset [Che13]. However, other models might perform equally well for the purpose of text comparison. Moreover, custom word vectors trained on a large domain-specific dataset, e.g. the Wikipedia encyclopedia for the Wikipedia revision comparison, could potentially yield even better results. 
+
+word2vec
+~~~~~~~~
+
+word2vec is a tool developed by Mikolov, Sutskever, Chen, Corrado, and Dean at Google. The two model architectures in the C tool were made available under an open-source license [Mik13]. Gensim provides Python bindings for word2vec [Řeh10].
+
+Word vectors encode semantic meaning and capture many different degrees of similarity [Lev14]. word2vec word vectors can capture linguistic properties such as gender, tense, plurality, and even semantic concepts such as "is the capital city of". word2vec captures domain similarity while other more dependency-based approaches capture functional similarity. 
+
+In the word2vec vector space, linear algebra can be used to exploit the encoded dimensions of similarity. Using this, a computer system can complete tasks like the Scholastic Assessment Test (SAT) analogy quizzes, that measure relational similarity. 
+
```

Comment:
Thanks for the feedback!

I changed the = to the more appropriate ≈.

The examples are meant to illustrate that the similarities hold true for a variety of different cases.

I'm familiar with Goldberg's work. Random Indexing and other techniques capture similarity as well and could be used, but, as Goldberg points out, there is plenty of evidence that word2vec outperforms traditional methods. 
word2vec is also very nicely integrated into the Python ecosystem.

t-SNE minimizes the Kullback-Leibler divergence between the
joint probabilities of high-dimensional data and a low-dimensional representation. The Kullback-Leibler divergence measures the dissimilarity ("distance") of two probability distributions by a discrete scalar.

I added that unsupervised methods are hard to evaluate, reinforced that the purpose of the paper is to illustrate a certain use case and provided more examples to make this clear. 

Thanks again,
Hendrik